sqlglot/transforms.py

```diff
@@ -368,6 +362,24 @@ def _udtf_type(u: exp.Unnest, has_multi_expr: bool) -> t.Type[exp.Func]:
                 expression.args["joins"].remove(join)
 
                 alias_cols = alias.columns if alias else []
+
+                """
+                Handle Presto CROSS JOIN UNNEST to LATERAL VIEW EXPLODE for Multiple or No Exploded table column alias.
+                
+                Spark/Hive LATERAL VIEW EXPLODE requires only single alias for respective exploded array/struct to be given for unnest ulike trino/presto which can take multiple.
+                https://cwiki.apache.org/confluence/display/Hive/LanguageManual+LateralView
+                https://spark.apache.org/docs/latest/sql-ref-syntax-qry-select-lateral-view.html
+                
+                """
+                # Replace multiple alias for single EXPLODE column with single static alias name: `t_struct`
+                if not has_multi_expr and (len(alias_cols) != 1):
+                    alias_cols = ["t_struct"]
+
+                    # [Optional] Do Update the Column reference in AST for table with current alias
+                    for column in expression.find_all(exp.Column):
+                        if alias and column.table == alias.name:
+                            column.set("table", "t_struct")
```

Comment:
Why is this optional? Wouldn't changing the alias require renaming the references? 

Also, if `alias` is `False` here the `find_all` would run for no reason, it's better to move it outside of the `for` loop since it's not changed.

Reply:
updated with this commit [52b41a0](https://github.com/tobymao/sqlglot/pull/4313/commits/52b41a0c3860ebb07fdde2842752e141d7d20ad2)